util/util.go

```diff
@@ -9,8 +9,8 @@ import (
 	"strings"
 	"time"
 
-	"github.com/apcera/libretto/ssh"
-	lvm "github.com/apcera/libretto/virtualmachine"
+	"github.com/avnish30jn/libretto/ssh"
```

Comment:
I am not sure we need to change this .. If it is local then we need to modify it to be 

apporbit/libretto/ssh and that should work..

Either way please change it to github.com/apporbit/libretto/ssh instead of your fork name.